### PR TITLE
Carlos

### PR DESCRIPTION
**Pull request by Carlos-Nisum**

### CODE REVIEW

**Review by GottaGitGoing — APPROVED** [review]
Looks good, just need to delete the example they have in app.component.html (so everything after your code Line 9) since those are there by default

File: .gitignore [hunk 1]
```diff
@@ -36,6 +36,7 @@ yarn-error.log
 /libpeerconnection.log
 testem.log
 /typings
+/marketplace-frontend
```
Comment (Yuteng-Li) [inline .gitignore]: is this line necessary to add since there is no more "marketplace-frontend" folder under the root of the repo?

**Review by Yuteng-Li — COMMENTED** [review]
can you review the .gitignore file and see if the change is intentional?

File: .gitignore (hunk 1 above)
Comment (Carlos-Nisum) [reply .gitignore]: Not necessary, just had that there because I accidentally had it in my project.

**Review by Andrew-Nisum — APPROVED** [review]

**Pull request merged**